http/vibe/http/common.d

```diff
@@ -336,13 +337,222 @@ class HTTPStatusException : Exception {
 	string debugMessage;
 }
 
+final class MultiPartBodyPart
+{
+	import vibe.stream.memory : createMemoryStream;
+
+	InetHeaderMap headers;
+	InputStream content;
```

Comment:
This needs to be `InterfaceProxy!InputStream` to enable forward compatibility with vibe-core.